news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Halloween 2023 Descends"
+date: 2023-10-22 14:00:00 +0000
+---
+
+By the ineffable will of the Dean "peppy" Herbert, we are having Halloween right this very moment — so come check out the winners from the Halloween 2023 Fanart Contest, soon to be featured in a game client near you!
+
+![](https://assets.ppy.sh/contests/186/banners/mocapixel.jpg)
```

Comment:
surely this one can be cropped a little better?

![image](https://github.com/ppy/osu-wiki/assets/36758269/fc323d73-7613-4580-b38d-fa5e602e14bc)

Reply:
using your crop from #osu-wiki, purged s3 so should show up soon